v21.2/use-changefeeds.md

```diff
@@ -98,9 +98,15 @@ The `kv.closed_timestamp.target_duration` [cluster setting](cluster-settings.htm
 
     To compare two different rows for [happens-before](https://en.wikipedia.org/wiki/Happened-before), compare the "updated" timestamp. This works across anything in the same cluster (e.g., tables, nodes, etc.).
 
-    Resolved timestamp notifications on every Kafka partition can be used to provide strong ordering and global consistency guarantees by buffering records in between timestamp closures. Use the "resolved" timestamp to see every row that changed at a certain time.
+    The complexity with timestamps is necessary because CockroachDB supports transactions that can affect any part of the cluster, and it is not possible to horizontally divide the transaction log into independent changefeeds. For more information about this, [read our blog post on CDC](https://www.cockroachlabs.com/blog/change-data-capture/) and see the following section on [resolved timestamps](#resolved-timestamps).
 
-    The complexity with timestamps is necessary because CockroachDB supports transactions that can affect any part of the cluster, and it is not possible to horizontally divide the transaction log into independent changefeeds. For more information about this, [read our blog post on CDC](https://www.cockroachlabs.com/blog/change-data-capture/).
+### Resolved timestamps
+
+The resolved timestamp acts as a checkpoint confirming that all change events up to its timestamp have been emitted.
+
+The resolved timestamp can be used to remove [duplicates](#duplicates) from change notifications. In some cases, it may be preferable to silence duplicate messages (e.g. when a client cannot accept duplicate messages). The resolved timestamp provides a way of identifying which messages can be cleaned up.
```

Comment:
I think we should expand on this to say _how_ it can be used to remove duplicates ie once you get a resolved message, you can safely ignore any messages that come in with timestamp less than the resolved timestamp